physics/PBL/MYNN_EDMF/module_bl_mynn.F90

```diff
@@ -350,12 +351,12 @@ MODULE module_bl_mynn
 CONTAINS
 
 ! ==================================================================
-!>\ingroup gsd_mynn_edmf
-!! This subroutine is the GSD MYNN-EDNF PBL driver routine,which
+!>\ingroup gp_mynnedmf
```

Comment:
What is gp? can it be dumped?

Reply:
Short for "group"? I dunno. @mzhangw Can you shed any light on this?

Reply:
Yes, it is short for "group"